modules/nf-core/umicollapse/meta.yml

```diff
@@ -11,7 +11,7 @@ tools:
         (UMIs)/Random Molecular Tags (RMTs)."
       homepage: "https://github.com/Daniel-Liu-c0deb0t/UMICollapse"
       documentation: "https://github.com/Daniel-Liu-c0deb0t/UMICollapse"
-      tool_dev_url: "https://github.com/Daniel-Liu-c0deb0t/UMICollapse"
+      tool_dev_url: "https://github.com/siddharthab/UMICollapse"
```

Comment:
I think we should keep this to still point to the original one for now. If we don't get a response for a year or so, then we can take over permanently.

Reply:
I prefer to leave it, though. I don't think that changing the URL gives the impression of taking development out of Daniel's hands, yet clarifies what changes constitute the version 1.1.0. After all, there are relevant improvements that are not yet reflected in the original repository - if we don't point to your for fork, I think it could be (even more) confusing.

Reply:
Makes sense.